src/ApiResponse/AbstractApiJsonResponse.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Silktide\ProspectClient\ApiResponse;
+
+use Psr\Http\Message\ResponseInterface;
+use function GuzzleHttp\json_decode;
```

Comment:
Is this intentional? Don't even know what this does?

Reply:
It is. Guzzle throws an internal exception when errors occur, rather than passing back false and having to subsequentially call json_last_error